terminal-here.el

```diff
@@ -74,28 +80,75 @@ Typically this is -e, gnome-terminal uses -x."
   :group 'terminal-here
   :type 'string)
 
+(defcustom terminal-here-multiplexers
+  '("screen" "tmux")
+  "List of terminal emulators."
+  :group 'terminal-here
+  :type 'string)
+
+(defcustom terminal-here-multiplexer
+  (terminal-here-find-executable terminal-here-multiplexers)
+  "The terminal multiplexer to run inside a terminal."
+  :type 'string
+  :options '("screen" "tmux")
+  :group 'terminal-here)
+
+(defcustom terminal-here-multiplexer-new-session t
+  "If t launch a new `screen' session inside multiplexer.
+`tmux' can only attach to existing session.
+
+If f attach to a `screen' session inside multiplexer."
+  :type 'boolean
+  :group 'terminal-here)
+
+(defcustom terminal-here-multiplexer-flags nil
+  "Flags passed to multiplexer."
+  :type 'list
+  :group 'terminal-here)
```

Comment:
For consistency with `terminal-here-terminal-command` we should probably have a variable called `terminal-here-multiplexer-command` which contains both the flags and the binary name.